libs/barista-components/breadcrumbs/src/breadcrumbs-item.ts

```diff
@@ -20,11 +20,11 @@ import { Directive, ElementRef } from '@angular/core';
  * A breadcrumbs item that can be used within the `<dt-breadcrumbs>`.
  *
  *  @example
- *  <a dtBreadcrumbsItem href="/hosts">Hosts</a>
+ *  <a DtBreadcrumbsItem2 href="/hosts">Hosts</a>
  */
 @Directive({
-  selector: 'a[dt-breadcrumbs-item], a[dtBreadcrumbsItem]',
-  exportAs: 'dtBreadcrumbsItem',
+  selector: 'a[dt-breadcrumbs-item], a[DtBreadcrumbsItem2]',
+  exportAs: 'DtBreadcrumbsItem2',
```

Comment:
The discussion we had last time might have been a little bit confusing, and I'm sorry about that. 
The selector for the breadcrumbsItem should stay at `'a[dt-breadcrumbs-item], a[dtBreadcrumbsItem]'`
It should also be exported as `DtBreadcrumbsItem` still. 

The last change had you changing the classname in line 32, which what I was commenting on last.